force-app/testRecordGenerator/classes/TestFieldFunctions.cls

```diff
@@ -111,12 +119,28 @@ global class TestFieldFunctions {
     global class Today extends TestFieldFunctionWithParent {
 
         global Integer addDays;
+        global Integer addMonths;
+        global Integer addYears;
+        global Boolean toStartOfWeek;
+        global Boolean toStartOfMonth;
 
         global Object getValue(String fieldName, Object value) {
             Date returnValue = Date.today();
             if(addDays != null) {
                 returnValue = returnValue.addDays(addDays);
             }
+            if(addMonths != null) {
+                returnValue = returnValue.addMonths(addMonths);
+            }
+            if(addYears != null) {
+                returnValue = returnValue.addYears(addYears);
+            }
+            if(toStartOfWeek == true) {
```

Comment:
I think that it would be nice to evaluate toStartOfWeek first. Then it would be possible to shift to the second day of the week by specifying toStartOfWeek=true and addDays=1

Reply:
Really good idea, that makes a lot of sense. I've moved addDays, addMonths and addYears to after the start of week/month calls.